grafana/resource_dashboard.go

```diff
@@ -301,5 +319,11 @@ func normalizeDashboardConfigJSON(config interface{}) string {
 	delete(dashboardJSON, "id")
 	delete(dashboardJSON, "version")
 	j, _ := json.Marshal(dashboardJSON)
-	return string(j)
+	sha256_store := os.Getenv("GRAFANA_CONFIG_JSON_SHA256")
+	if sha256_store == "yes" {
+		configHash := sha256.Sum256([]byte(j))
```

Comment:
`j` already is a `[]byte`:
```suggestion
		configHash := sha256.Sum256(j)
```